common/src/main/java/com/github/khanshoaib3/minecraft_access/mixin/BookEditScreenMixin.java

```diff
@@ -37,57 +41,77 @@ public class BookEditScreenMixin {
     @Shadow private ButtonWidget doneButton;
 
     @Shadow private String title;
-    boolean firstTimeInSignMenu = true;
-    String previousContent = "";
+
+    @Unique boolean minecraft_access$firstTimeInSignMenu = true;
```

Comment:
What does the `@Unique` does? Is it like unique for each instance or object of the class?

Reply:
> What does the `@Unique` does? Is it like unique for each instance or object of the class?

That's a mark for notifying Mixin DO NOT override them onto vanilla code.
After I upgraded some plugins in IntelliJ, these linter  (suggestions) poped up in MIxin classes.